src/Sarif.Driver/Sdk/Skimmer.cs

```diff
@@ -51,7 +49,7 @@ public Skimmer()
 
         public override MultiformatMessageString FullDescription => throw new InvalidOperationException($"The {nameof(FullDescription)} property must be overridden in the SkimmerBase-derived class.");
 
-        public override MultiformatMessageString ShortDescription => new MultiformatMessageString { Text = ExtensionMethods.GetFirstSentence(FullDescription.Text) };
+        public override MultiformatMessageString ShortDescription => null;
```

Comment:
>null [](start = 69, length = 4)

is this right? if we see the tests, we removed all short descriptions

Reply:
Yes, this is right. In SARIF, this null return value will indicate that the short description isn't populated (in the default implementation) and it will be omitted from the SARIF log file entirely (which is correct). In SARIF, the very first sentence of the full description serves as the concise short description, if it exists. Rule authors who follow this pattern don't need to provide a short description (and in fact this is the recommended practice).

---
In reply to: [532184727](https://github.com/microsoft/sarif-sdk/pull/2169#discussion_r532184727) [](ancestors = 532184727)